package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/classwrap.js",
   "jsnext:main": "src/index.js",
   "module": "src/index.js",
+  "typings": "src/classwrap.d.ts",
```

Comment:
@Swizz What's this? First time I see this.

Reply:
This is the way to tell all typescript compliant tools where to find your DTS.
https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

You can either use `types` or `typings` both work.

Reply:
Thanks.